js/blogtini.js

```diff
@@ -395,10 +430,17 @@ async function storage_create() { // xxx
 
       // now make the requests in parallel and wait for them all to answer.
       const vals = await Promise.all(proms)
-      const file2markdown = files.reduce((obj, key, idx) => ({ ...obj, [key]: vals[idx] }), {})
+
+      /**
+       * documentResponseObjects: They're a collection of HTTP Response objects.
+       *
+       * That's where we can get the HTTP Response of files,
+       * maybe we should keep the response headers and use in storage.
+       */
+      const documentResponseObjects = files.reduce((obj, key, idx) => ({ ...obj, [key]: vals[idx] }), {})
 
       // eslint-disable-next-line no-use-before-define
-      await parse_posts(file2markdown)
+      await parse_posts(documentResponseObjects)
```

Comment:
I got surprised to realize that those are a collection of HTTP Response objects.